README.md

```diff
@@ -4,10 +4,17 @@
   <a href="https://github.com/actions/setup-ruby"><img alt="GitHub Actions status" src="https://github.com/actions/setup-ruby/workflows/Main%20workflow/badge.svg"></a>
 </p>
 
-This action sets up a ruby environment for use in actions by:
+This action sets up Ruby for use in actions by adding a version of Ruby already existing in the toolcache to the PATH.
```

Comment:
Can you also explain what the toolcache is?

Reply:
I added a sentence about it.